.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build:
+  test-all:
```

Comment:
What's the current purpose of this file?

Reply:
This might've been a `rebase` misshap. This file was migrated to `base_tests.yml`.